editor/src/project/add/mesh.ts

```diff
@@ -88,6 +88,10 @@ export function addGroundMesh(editor: Editor, parent?: Node) {
 		width: 1024,
 		height: 1024,
 		subdivisions: 32,
+		heightMapTexture: null,
+		minHeight: 0,
+		maxHeight: 10,
+		useHeightMap: false,

```

Comment:
I think this can be determined to "true" when heightMapTexture is not null?

Reply:
I was doing that the user experience is amazing... but when I had lots of subdivisions it was veeeery slow. My PC can handle but some PCs might have a hard time processing it.